web_docs/index.md

```diff
@@ -259,3 +274,155 @@ After the spacecraft has received enough contact time to download a data file qu
 [Capture an signal using the SDR payload and download a compressed IQ file](https://github.com/nsat/space-services-user-guide/tree/main/examples/software_in_space/compress_iq_file/README.md) -
 A trivial example showing the code necessary to capture a signal using the SDR payload, compressing the resulting IQ file, and downloading the compressed file from the 
 payload to S3.
+
+
+## Tools & Utilities
+The following tools and utilities are developed and supported by Spire, and provided on the applicable payload(s) for
+customers to run directly or indirectly within a payload windows [`user_command`](/tasking-api-docs/index.html#user_command) parameter.
```

Comment:
**Question**: Will this link work correctly when on the website? It doesn't work when clicking through on GitHub.

Reply:
Works on my machine. What do you mean by "doesn't work"? Do you mean that the page doesn't load, or that it doesn't highlight the right section?

Reply:
Ok, I think because I was checking by letting GitHub render the markdown, this comment is irrelevant.